userspace/libscap/engine/udig/CMakeLists.txt

```diff
@@ -1,3 +1,6 @@
 include_directories(${LIBSCAP_INCLUDE_DIRS})
 add_library(scap_engine_udig scap_udig.c)
-target_link_libraries(scap_engine_udig scap_engine_util rt)
+target_link_libraries(scap_engine_udig scap_engine_util)
+if(UNIX AND NOT APPLE)
+    target_link_libraries(scap_engine_udig rt)
```

Comment:
`rt` is only available on linux.

Reply:
What about windows?

Reply:
windows and macos do not need `rt` because their stdlib already provides the symbols

Reply:
Sorry I was not clear, I meant maybe we can add also `AND NOT WIN32` to the if statement just to be clear, WDYT?

Reply:
Well there is the `if(UNIX`  :D 
pretty self-explanatory ahah